javascript-data-model/order-history.js

```diff
@@ -0,0 +1,31 @@
+const orderHistory = [
+  {
+    items: ['JavaScript for impatient programmers'],
```

Comment:
Nice job with this! Though putting _all_ properties as arrays may get cumbersome to deal with when sorting through it all.

Another way to do it is to have an `items` property that has an array for each `item` with it's own properties, while the `order` object has the main properties like `totalPrice`, `orderedBy`, `orderDate`, etc.

I'll go over it in Q&A

Reply:
> Nice job with this! Though putting _all_ properties as arrays may get cumbersome to deal with when sorting through it all.
> 
> Another way to do it is to have an `items` property that has an array for each `item` with it's own properties, while the `order` object has the main properties like `totalPrice`, `orderedBy`, `orderDate`, etc.
> 
> I'll go over it in Q&A

Thank you for the feedback Kurt! :)